src/clickhouse_migrations/cmd.py

```diff
@@ -13,6 +14,11 @@
     MIGRATIONS_DIR,
 )
 
+def log_level(value: str) -> str:
+    if value.upper() in ("DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"):
```

Comment:
maybe it's better to use logging._nameToLevel.keys()?
yes, it's private variable, but it contains all levels and aliases.